ethcore/builtin/src/lib.rs

```diff
@@ -548,14 +572,39 @@ impl Bn128Pairing {
 	}
 }
 
+impl Implementation for EIP1962 {
+	/// Can fail in many cases, so leave it for an implementation
+	fn execute(&self, input: &[u8], output: &mut BytesRef) -> Result<(), &'static str> {
+		let result = eth_pairings::public_interface::API::run(&input);
+		if result.is_err() {
+			trace!("EIP1962 error: {:?}", result.err().expect("is error"));
+			return Err("Precompile call was unsuccessful");
+		}
+		let result: Vec<u8> = result.expect("some result");
```

Comment:
All `expect`s should motivate the reason why it is safe to unwrap, e.g. "we just checked if is_err so we know it's ok; qed"